### PR TITLE
Source Kyriba: Updates CDK, Increases Testing Coverage, Fixes Acceptance Test Config, Adds Expected Records

### PR DESCRIPTION
**Pull request by pnilan**
## What
- Updates CDK to latest version
- Increases test coverage
- Adds additional error handling for invalid config
- Updates `acceptance-test-config` for all tests to run
- Adds expected records for all streams
- Sets `additionalProperties` to true for all streams
- Sets `fails_on_extra_column` to false in `acceptance_test_config`
- Adds missing fields to `cash_flows` and `bank_balances_intraday` streams
- Cleans up schemas to remove extra validation properties
- Migrates to new base image
- Updates Kyriba.md documentation to conform to current standard

### CODE REVIEW

File: airbyte-integrations/connectors/source-kyriba/source_kyriba/source.py [hunk 1]
```diff
@@ -270,9 +270,15 @@ def gateway_url(self, config: Mapping[str, Any]) -> str:
         return f"https://{config['domain']}/gateway"
 
     def check_connection(self, logger, config) -> Tuple[bool, any]:
-        client = KyribaClient(config["username"], config["password"], self.gateway_url(config))
-        client.login()
-        return True, None
+        try:
+            client = KyribaClient(config["username"], config["password"], self.gateway_url(config))
```
Comment (artem1205) [inline airbyte-integrations/connectors/source-kyriba/source_kyriba/source.py:274]: Kyriba uses Oauth2 Protocol for authentication (access token will expire in 1 hour).
Do we need to migrate it as well?

Comment (pnilan) [reply airbyte-integrations/connectors/source-kyriba/source_kyriba/source.py:274]: @artem1205 This is a community connector with few active users -- is the current authentication method inadequate?

Comment (artem1205) [reply airbyte-integrations/connectors/source-kyriba/source_kyriba/source.py:274]: >  is the current authentication method inadequate?

Well, it does work but the token we obtain in `login` method will work only during **_1 hour_**. After that we need to exchange it to the new one.

Comment (artem1205) [reply airbyte-integrations/connectors/source-kyriba/source_kyriba/source.py:274]: agreed, this will work

**Review by artem1205 — APPROVED** [review]
@pnilan , From description
> Updates Docker build Python version to 3.10

We can get rid of `Dockerfile` and use our base image docker.io/airbyte/python-connector-base

**Pull request merged**